README.md

```diff
@@ -151,24 +155,24 @@ The react app in this quickstart shows us an example of the username/password au
 
 Sometimes, you would want to upload some files to the cloud. This can range from a profile pic for your user or images for things listed on your app. You can securely add, remove, manage, update files such as pictures, videos, documents using Hasura filestore.
 
-You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/users/index.html)
+You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/files/index.html)
 
 The react app in this quickstart shows us an example of uploading a file to the filestore. To see it in action navigate to `https://ui.cluster-name.hasura-app.io/filestore` and take a look at `filestore.js` for the code.
 
 ## Custom Service
 
-There might be cases where you might want to perform some custom business logic on your apis. For example, sending an email/sms to a user on sign up or sending a push notification to the mobile device when some event happens. For this, you would want to create your own custom service which does these for you on the endpoints that you define.
+There might be cases where you might want to perform some custom business logic on your APIs. For example, sending an email/sms to a user on sign up or sending a push notification to the mobile device when some event happens. For this, you would want to create your own custom service which does these for you on the endpoints that you define.
 
-This quickstart comes with one such custom service written in `nodejs` using the `express` framework. Check it out in action at `https://app.cluster-name.hasura-app.io` . Currently, it just returns a "Hello-React" at that endpoint.
+This quickstart comes with one such custom service written in `nodejs` using the `express` framework. Check it out in action at `https://api.cluster-name.hasura-app.io` . Currently, it just returns a "Hello-React" at that endpoint.
```

Comment:
We're using `www` as the subdomain in the hello-world quickstart. You can change it to that?